packages/flutter/test/gestures/velocity_tracker_test.dart

```diff
@@ -42,7 +42,7 @@ void main() {
         tracker.addPosition(event.timeStamp, event.position);
       }
       if (event is PointerUpEvent) {
-        _checkVelocity(tracker.getVelocity(), expected[i]);
+        expect(_checkVelocity(tracker.getVelocity(), expected[i]), isTrue);
```

Comment:
Not necessary needed to merge this pr, we may also consider move the expect to _checkVelocity function directly.